pkg/controller/deployment/util/deployment_util.go

```diff
@@ -606,6 +606,9 @@ func EqualIgnoreHash(template1, template2 *v1.PodTemplateSpec) bool {
 	// Remove hash labels from template.Labels before comparing
```

Comment:
we should also update this function in `kubectl/pkg/polymorphichelpers/rollback.go`